Modrinth/index.json

```diff
@@ -0,0 +1,440 @@
+{
+  "formatVersion": "1",
+  "game": "minecraft",
+  "versionId": "1.9.1",
+  "name": "Fabulously Optimized",
+  "summary": "A Fabric-based modpack for Minecraft that focuses on performance and graphics optimizations",
+  "files": [
+    {
+      "path": "mods/sodium-fabric-mc1.16.3-0.1.0.jar",
+      "hashes": {
+        "sha512": "2597064f7116315adcfe114f0d76c04233791fed049cbbcf722f878002bf14cd8dec421806da639f31ffd496b9dedac9e39e2b48f43007427fa0f659ec5118e2"
+      },
+      "env": {
+        "client": "required",
+        "server": "unsupported"
+      },
+      "downloads": [
+        "https://cdn.modrinth.com/data/AANobbMI/versions/mc1.16.3-0.1.0/sodium-fabric-mc1.16.3-0.1.0.jar",
```

Comment:
> It will prioritise Modrinth links over GitHub Release links over CurseForge links.

Hmm. If Modrinth links would be any easier to manage in a Modrinth modpack, sure. But otherwise I think I'd prefer CF links the most to give credit where due (increase downloads) and also give the developers the opportunity to receive reward points.

Reply:
I was under the impression that the download counter only incremented and reward points only given if it was downloaded via the frontend, not via the cdn, but I could be wrong. Plus, as mentioned in one of my earlier comments, the purpose of this was mostly for testing on Modrinth. Every mod except Hydrogen has a CurseForge link, but that's not the case the other way around.

Reply:
Thinking back, you're right. Plus Modrinth supposedly plans to launch its own reward system, so this point is moot :P

Reply:
I'll be changing the preference to just have whatever the packwiz link is, to reduce upkeep.